bench/src/benchmarks/mod.rs

```diff
@@ -1,3 +1,20 @@
+use iggy::client::UserClient;
+use iggy::clients::client::IggyClient;
+use iggy::users::login_user::LoginUser;
+
 pub mod poll_messages_benchmark;
 pub mod send_and_poll_messages_benchmark;
 pub mod send_messages_benchmark;
+
+const ROOT_USERNAME: &str = "iggy";
```

Comment:
maybe define it once in sdk as 
`DEFAULT_ROOT_USERNAME` and use it this way everywhere?

Reply:
Ok